### DISCUSSION TITLE
How to do multipass ?

### DISCUSSION THREAD

**Discussion by NicoPy** [arkypita/LaserGRBL#1635] [Q&A] [answered]
Hi,

What is the correct way to do multi-pass (for cutting) ?

Regards,
Nicolas

**Comment by Tribo-2dot0** [comment, marked as answer]
I think maybe it's here...
![Captura de Tela (79)](https://user-images.githubusercontent.com/83982525/139565862-9a5dedc3-e509-4546-95c1-186523494679.png)

**Comment by NicoPy** [comment]
@Tribo-2dot0 Ah, I didn't noticed it. Thanks.
I never seen any documentation/video/tutorial mentioning it. Even on the [official help page](https://lasergrbl.com/usage/user-interface/), it is not there. Maybe it should be good to add it.

One more question : Does it affect all added files ? 
I mean, when engraving and cutting, you _Open_ a file for the engraving pass then you _Add_ a file for the cutting pass. I guess the _Perform multiple passes_ control affects all files (so it is not usable in this scenario). Right ?

Note : I still don't have my engraver, so I can't do any test for now.

**Comment by StuartB4** [comment]
It is in the documentation at the bottom of this page. 
https://lasergrbl.com/usage/load-and-send
But not the part about it affecting all added files.
But it would do multiple passes on all the loaded files.

There is a way around it though. Like this.

![Unt111111itled](https://user-images.githubusercontent.com/16980026/139584656-d074b2af-d000-4f4a-8cf1-141f7e61d85c.png)

![Unt11112211itled](https://user-images.githubusercontent.com/16980026/139585520-a27c7f62-eac5-44a5-bf70-d16664e4c905.png)

**Comment by NicoPy** [comment]
Thanks for your great explanations.
